flink-tests/src/test/scala/org/apache/flink/api/scala/runtime/CaseClassComparatorTest.scala

```diff
@@ -34,7 +34,7 @@ import org.mockito.Mockito
 
 class CaseClassComparatorTest {
 
-  case class CaseTestClass(a: Int, b: Int, c: Int, d: String)
+  case class CaseTestClass(a: Int, b: Int, c: Int, d: String, e: Option[Int])
```

Comment:
Please do not change this class but create a new `OptionSerializerTest` which extends `SerializerTestBase`. See for instance `IntSerializerTest` for how the test base is used.